stripe/res/layout/activity_payment_sheet.xml

```diff
@@ -5,76 +5,74 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <androidx.constraintlayout.widget.ConstraintLayout
+    <LinearLayout
```

Comment:
Was there a problem with `ConstraintLayout`?

Reply:
Yes, for some reason it was adding a padding on the top. Since all children are laid out linearly I changed to LinearLayout.

Reply:
I imagine the `clipToPadding` change may have been the reason?

Reply:
`LinearLayout` is fine for what it's worth, was mainly curious.